README.md

```diff
@@ -145,7 +145,7 @@ When a shell script is run from an SSH session, as soon as the pipe is broken an
     `kill %<job_number>`
 
 5. If you want the job to continue running even after you log out, you can `disown` it after starting it:  
-    `disown -h %1  # Replace 1 with the job number if different`
+    `disown %1  # Replace 1 with the job number if different`
```

Comment:
IMO the -h parameter is necessary so that the process is not killed when the connection drops
[https://linuxcommand.org/lc3_man_pages/disownh.html](url)

Reply:
Interesting. This is what grok has to say on the issue:

The discrepancy in behavior regarding the `-h` parameter of the `disown` command between Bash and Zsh on Ubuntu LTS 22.04 can be attributed to differences in how these shell environments implement job control and the `disown` command. Here's a breakdown:

### Bash (Bourne Again Shell)

In Bash, the `-h` option for `disown` indeed removes the job from the shell's job table, stopping the job from receiving a `SIGHUP` when the shell exits, and making it invisible to the `jobs` command. This behavior is straightforward and as per the man page or Bash documentation.

### Zsh (Z Shell)

Zsh, while very similar to Bash in many respects, has its own implementation of commands and features, often with differences to enhance user experience or for compatibility with other Unix shells:

- **Zsh's `disown`**: In Zsh, the `disown` command has different options:

  - `-h` or `--ignore-sighup`: This option in Zsh only affects how the job is handled regarding `SIGHUP`. It doesn't remove the job from the job list like it does in Bash.
  
  - `-a` or `--all`: If you want to remove a job from the job list in Zsh, you would typically use `-a` which has the effect similar to Bash's `-h`, but it's specifically for removing all jobs from the table.

  - To achieve the effect of Bash's `disown -h` in Zsh, you might need to use something like `disown -ar` where `-r` refers to removing the job, and `-a` indicates all, though in practice, you'd specify the job rather than all (`-ar %job_id`).

### Why the Difference?

- **Historical and Design Choices**: Zsh was designed to be more extensible and to some extent, more consistent across different Unix-like systems. Different options for the same command might reflect historical development paths or design decisions to offer more control or different default behaviors.

- **Compatibility and Expansion**: Zsh often expands or changes the behavior of commands to offer more functionality or to correct what might be seen as limitations or inconsistencies in Bash. This can lead to different option names or behaviors even for similar commands.

### Practical Advice:

- **For Bash users**: You can use `disown -h` as expected.

- **For Zsh users**: If you want to both stop `SIGHUP` and remove the job from the job list:
  - Use `disown -ar %job_id` (where `%job_id` is the job you want to disown).
  - Or, if you want to remove all jobs and make them not receive `SIGHUP`, you might use `disown -ar` without specifying a job.

- **Switching Shells**: If you're frequently moving between Bash and Zsh, remember these differences or consider using aliases or shell functions to map similar behaviors across shells.

Understanding these nuances helps in navigating shell differences effectively, ensuring you can manage job control consistently across different environments.

Reply:
I just did `disown %1`. That worked and survived exiting my terminal/shell.  `nohup ./minilaunch.sh` which also works, but you've got to `disown` that as well.

Reply:
I will update the README to differentiate between bash and zsh and hence close the PR